posts/views.py

```diff
@@ -428,6 +437,120 @@ def get(self, request, authorid):
                 contentTypes.append( "<p>" + post.content +"</p>")
         return render(request, 'author/author_posts.html', context={'author': author, 'posts':serializer.data, 'contentTypes':contentTypes})
 
+class GetAuthorPosts(views.APIView):
+    authorHelper = FrontEndAuthorPosts()
+
+    def get(self, request, authorid):
+        paginator = CustomPagination()
+        author = self.authorHelper.get_user(authorid)
+        user = request.user
+        friendship_level = self.authorHelper.get_friendship_level(request, author)
+        posts = self.authorHelper.get_feed(user, author, friendship_level)
+        result_page = paginator.paginate_queryset(posts, request)
+        serializer = PostSerializer(result_page, many=True)
+        return paginator.get_paginated_response(serializer.data, "posts")
+
+class FrontEndAuthorPosts(TemplateView):
```

Comment:
I think this should be moved to the feed view file just because it has to do with the author feeds.